### DISCUSSION TITLE
Parallel MEEP on AWS ParallelCluster for Adjoint Optimization?

### DISCUSSION THREAD

**Discussion by satadrudas** [NanoComp/meep#2566] [Q&A] [answered]
I am currently trying to create a dataset of inverse designed device initialized with random permittivity distribution in the design region. Needless to say that the the computing resource is very intensive and unfortunately I don't have access to a computing cluster.

I am planning to use some free trial versions of on clusters such as AWS, Deep Ocean etc. (Any other suggestions are welcome)

I wanted to know if anyone used any on the cloud services for cluster computing?

**Comment by smartalecH** [comment, marked as answer]
I've used AWS for meep computations (including optimizations) extensively. I know many others have too. You can probably get some academic credits. 

Most major cloud vendors would be a good option (GCP, Azure, AWS, etc).

**Reply by satadrudas** [reply]
Thats great !! Thanks

**Reply by satadrudas** [reply]
Hi again. While I am trying to navigate through the various EC2 instances of AWS, is there any specific EC2 instances that you would suggest for adjoint optimization?

**Reply by smartalecH** [reply]
Every problem is different. Stick with those with high memory bandwidth, as that's what bottlenecks FDTD. In essence, you just need to try a few.
